mod-data-import-converter-storage-server/src/main/java/org/folio/dao/AbstractProfileDao.java

```diff
@@ -185,6 +186,80 @@
     return future;
   }
 
+  @Override
+  public Future<Boolean> markProfileAsDeleted(String profileId, String tenantId) {
```

Comment:
plz update java doc in interface with better description about how this method is work and what it do